index.html

```diff
@@ -208,23 +213,24 @@ <h2 class="title">About<br>Myself</h2>
                                     <img src="./assets/pictures/Frame_css.png" alt="">
                                 </li>
                             </ul>
+                            
                         </div>
-                        
+                        <img src="./assets/pictures/Language_List.png" alt="" id="lang_list">
                         <div class="union">   
                             <a class="words">Frameworks</a>
-                            <img src="./assets/icons/union_geater.png" alt="">
+                            <i id="angle_" class="fa-solid fa-angle-right"></i>
                         </div>    
                         <hr>
                         <div class="union">    
                             <a class="words">Skills</a>
-                            <img src="./assets/icons/union_geater.png" alt="">
+                            <i id="angle_1" class="fa-solid fa-angle-right"></i>
                         </div> 
                         <hr>                        
                     </div>                   
                 </div> 
             </section>   
             <!--Contac form starts here-->
-            <section class="contact_me">
```

Comment:
- [ ] Contact form is almost there! There's still room to make it even more similar to the template:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-02 a la(s) 10 40 04 p  m](https://user-images.githubusercontent.com/83717285/171784756-1a40d453-68de-4896-94aa-67af4ce6e274.png)|![Captura de pantalla 2022-06-02 a la(s) 10 40 13 p  m](https://user-images.githubusercontent.com/83717285/171784807-83b66181-6da1-4a92-bde1-9072f3099f67.png)|

- [ ] It would be great if you reduced the height and width of the form fields.

- [ ] Let's make the button smaller, like in the Figma template.

- [ ] I would suggest increasing the space between the input fields.